### DISCUSSION TITLE
[Thesis] Player Analysis

### DISCUSSION THREAD

**Discussion by cagix** [Programmiermethoden/Dungeon#1380] [Ideas]
Bestimmung der Spielertypen (mit AI?) über kleine Aufgaben, Adaption im Spiel

Variationen:
- Spielertypen: Explorer, Achiver, Socializer, ... (nach [Bartle](https://www.interaction-design.org/literature/article/bartle-s-player-types-for-gamification)) - Anpassen der genutzten Spielmechanismen
- Analyse der Bearbeitung der Aufgabe(n) - Vorschlagen ergänzender oder neuer/anderer Aufgaben (Level/Quests)

**Comment by AMatutat** [comment]
Könnte sich gut mit https://github.com/Programmiermethoden/Dungeon/issues/193 kombinieren lassen. 
Ich kriege Aufgaben basierend auf meinen Fachlichenskill und in der Auführung basierend auf meine Spielerischen Vorlieben.
